testing/src/scenario/_ops_main_mock.py

```diff
@@ -15,7 +15,7 @@
     Any,
     Generic,
     Sequence,
-    cast,
+    cast
```

Comment:
This change isn't right.

```suggestion
    cast,
```

Please run `tox -e format` and `tox -e lint`.